Feature/site step 1

Horomi-103 opened the pull request:
На проверку

Commit:
Сделала верхнее меню. Доделать!

Commit:
Не разобралась, как сделать текст посередине картинки

Commit:
проблемы с выравниванием текста по центру картинки

Commit:
картинки и 2 стр

Commit:
Поправила левое меню. Исправить правое. Добавить 2 блока.

Commit:
Попавила правое меню. Нужно доработать

Commit:
Изменила id на class. Перенесла стили в css. Создала папку для картинок

Commit:
Сделала меню на все 3 стр. Поправила стили под классы.

crutch12 commented:
К названиям коммитов нету претензий, тут всё +- норм, молодец!

crutch12 commented:
1) никаких русских букв в названиях файлов/папок. И (если есть) в коде - тоже. Русскими только комментарии и контент могут быть
2) главную страницу обзови index.html, тогда она автоматически будет открываться при открытии сайта, выложенного на gh pages
3) выложи на gh pages

crutch12 commented on summary.html:
```diff
@@ -0,0 +1,83 @@
+
+        <div class="cont2">
+            <ul>
+                <li><a href="#"><img src="Картинки/stat.png">
```
странное форматирование, не надо так вкладывать элементы
лучше каждый вложенный элемент открывать на новой строчке

crutch12 commented on summary.html:
```diff
@@ -0,0 +1,83 @@
+            <ul>
+                <li><a href="#"><img src="Картинки/stat.png">
+                        <p>Statistics</p>
+                    </a></li>
```
ну и то же самое с закрытием - не надо их на одной строке закрывать

crutch12 commented on map.html:
```diff
@@ -0,0 +1,63 @@
+<html>
+
+<head>
```
head и body вложены в <html>, так что у них тоже должны быть отступ

crutch12 commented on common.css:
```diff
@@ -0,0 +1,111 @@
+    height: 10%;
+}
+
+.cont1 {
```
название ни о чем не говорит. должно быть понятным, сейчас я не понимаю, за что этот класс отвечает

crutch12 commented on common.css:
```diff
@@ -0,0 +1,111 @@
+    width: 50%;
+    float: left;
+}
+.m {
```
то же самое - проблема с именем

crutch12 commented on common.css:
```diff
@@ -0,0 +1,111 @@
+    height: 7%;
+}
+
+.men1 {
```
то же самое - проблема с именем

crutch12 commented on common.css:
```diff
@@ -0,0 +1,111 @@
+    margin-left: 3%;
+    float: left;
+}
+.mp {
```
то же самое - проблема с именем

crutch12 replied:
и все внизу - тоже

crutch12 commented on common.css:
```diff
@@ -0,0 +1,111 @@
+    width: 50%;
+    float: left;
+}
+.not {
```
Не надо называть класс `not`
В css есть селектор `:not`, ты с ним пересекаешься по имени и это путает, поэтому переименуй

crutch12 commented on sum.css:
```diff
@@ -0,0 +1,67 @@
+.prof, .navig {
+    display: inline-block;
+}
+.prof {
```
то же самое - проблема с именем

Commit:
1. Убрала русский язык в названиях
2. Переименовала классы
3. Переделала стили. Размеры оставила в %, ибо в все поплыло
